astro/src/content/blog/announcing-fusionauth-1-57.mdx

```diff
@@ -0,0 +1,33 @@
+---
+publish_date: 2025-04-03
+title: Announcing FusionAuth 1.57.0 - The Webhook Wombat
+description: Whether you’re processing massive numbers of webhooks or simplifying event management, version 1.57.0 improves system performance.
+authors: Brad McCarty
+image: /img/blogs/release-1-57/fusionauth-1-57.png
+categories: Product
+tags: SSO, Custom Domains, API Key Security
+excerpt_separator: "{/* more */}"
+---
+
+We’re happy to release the latest version of FusionAuth into the wild. The 1.57.0 update focuses on improving webhook performance and usability, so that your integrations run smoother than ever.
+
+We’ve dubbed 1.57.0 The Webhook Wombat. Our furry little friend has been digging deep to streamline the webhook functionality and to clean up your event logs faster than ever before.
+
+![The webhook wombat digs deep for out of date logs](/img/blogs/release-1-57/webhook-wombat.png)
+
+## Webhook Enhancements
+
+We released webhook event logs back in \[1.53.0\](/blog/announcing-fusionauth-1-53). These logs gave FusionAuth users a historical record of webhook events and their responses. This is useful for auditing and debugging purposes.
+
+With the release of 1.57.0, we’re doing some housekeeping to make sure that these events are logged and managed in a more efficient way.
+
+Here’s what’s new:
+
+- **Disable Webhook Event Logs**: New instances of FusionAuth will have logging disabled by default. If you have an existing instance and upgrade to 1.57.0, nothing changes. You’ll retain your current settings. If you have a new instance and want to turn on logging, no problem, use our API or admin UI to do so.
+- **More Performant Reaper**: A webhook log reaper manages and potentially deletes old or unnecessary webhook event logs to prevent possible storage issues and maintain system efficiency. Our goal here was to help keep your cleanup process efficient, regardless of size.
```

Comment:
```suggestion
- **More performant reaper**: A webhook log reaper manages and potentially deletes old or unnecessary webhook event logs to prevent possible storage issues and maintain system efficiency. Our goal here was to help keep your cleanup process efficient, regardless of size.
```